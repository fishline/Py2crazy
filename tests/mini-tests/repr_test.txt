print repr("aoooooooooga")
x = 'bahhhhhhh'
print `x`
