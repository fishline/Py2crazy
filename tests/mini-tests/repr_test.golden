Disassembling top-level module in ./mini-tests/repr_test.txt
  1       0 LOAD_NAME            print [43m[30mrepr[39m[49m("aoooooooooga")
          3 LOAD_CONST           print repr([43m[30m"aoooooooooga"[39m[49m)
          6 CALL_FUNCTION        print repr([43m[30m"aoooooooooga"[39m[49m)
          9 PRINT_ITEM           print repr([43m[30m"aoooooooooga"[39m[49m)
         10 PRINT_NEWLINE        print repr([43m[30m"aoooooooooga"[39m[49m)

  2      11 LOAD_CONST           x = [43m[30m'bahhhhhhh'[39m[49m
         14 STORE_NAME           [43m[30mx[39m[49m = 'bahhhhhhh'

  3      17 LOAD_NAME            print `[43m[30mx[39m[49m`
         20 UNARY_CONVERT        print [43m[30m`x`[39m[49m
         21 PRINT_ITEM           print [43m[30m`x`[39m[49m
         22 PRINT_NEWLINE        print [43m[30m`x`[39m[49m
         23 LOAD_CONST           print [43m[30m`x`[39m[49m
         26 RETURN_VALUE         print [43m[30m`x`[39m[49m
